brave/browser-laptop
# Add multiple selection for about:history

fix #3949

Auditors: @bsclifton, @bbondy

Test Plan:
For click behavior
---
1. First click should select a entry
2. All the following entries should be selectd with cmd/ctrl + click
3. Another click without cmd/ctrl should cancel selection

For shift click behavior
---
1. Click one entry
2. Shift + click another entry in the same table
3. The entry between these two will also be selected

For shift click + cmd/control click
---
1. Click one entry as start point
2. command/control click other entries between start point and end point in next step
3. Shift + click an entry as end point
4. The entry between start and end point should be selected

For multi selector
---
1. Open about:history
2. Multi-select entries
3. Test "Open Links in New Tabs"
4. Test "Open Links in New Private Tabs"
5. Test "Open Links in New Session Tabs"
6. Test "Delete History Entries"
